hydraw/README.md

```diff
@@ -28,8 +28,16 @@ The frontend is written in vanilla HTML/CSS/JS and is dynamically served from th
 
 All commands must be run from within `hydraw/` and assuming your Hydra API host+port is `hydra.example.io:4001` and your cardano signing key is `cardano.sk`:
 
+You can choose the network by specifying `HYDRAW_NETWORK`.
+
+This value contain either network magic number in case you want to run hydraw on one of the testnets or a string "mainnet".
+
+In case HYDRAW_NETWORK is not set we default to `Testnet (NetworkMagic 42)`.
+
+In case the magic number parsing fails we assume you want to run on Mainnet!
```

Comment:
Must: fix this explanation (it is not consistent with the ipmlementation in `hydraw/app/Main.hs`)

Should: default to the same default as the hydra-node uses. IIRC this is testnet 42, and not mainnet.

Reply:
I made the implementation and the readme match now.